<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bundle_selector</class>
 <widget class="QGroupBox" name="bundle_selector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Bundles</string>
  </property>
  <property name="title">
   <string/>
  </property>
  <property name="flat">
   <bool>true</bool>
  </property>
  <property name="checkable">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="cbTogleAll">
     <property name="text">
      <string>Select/Deselect All</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="lvBundles"/>
   </item>
   <item>
    <widget class="QPushButton" name="btnSave">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
